README.md

```diff
@@ -1,25 +1,25 @@
+One could defend our contracts with oil and gas by suggesting that our technology helps in their R&D efforts to develop sustainable energy. Yet, Azure Executive Tom Keane, in discussing [Microsoft's multimillion dollar contract with Chevron](https://news.microsoft.com/transform/chevron-fuels-digital-transformation-with-new-microsoft-partnership/), asked "How can we [..] more efficiently do oil exploration? Or more efficiently do sensor management from an offshore oil rig?" 
 
 Furthermore, Microsoft struck a deal last year with Norweigen petroleum company Equinor worth ["hundreds of millions of dollars"](https://www.marketscreener.com/EQUINOR-1413290/news/Equinor-and-Microsoft-in-Cloud-Services-Partnership-Deal-26794775/) and participated in an oil and gas conference with the goal of ["Empowering Oil and Gas with AI."](https://news.microsoft.com/en-xm/2018/11/12/microsoft-demonstrates-the-power-of-ai-and-cloud-to-oil-and-gas-players-at-adipec-2018/ ) Earlier this year, Microsoft announced a new contract with [Exxon Mobil](https://www.reuters.com/article/us-exxon-mobil-microsoft-cloud/exxon-microsoft-strike-cloud-computing-agreement-for-u-s-shale-idUSKCN1QB1N8
-), which the company boasts to be "the industry's largest in cloud computing." 
+), which the company says is "the industry's largest in cloud computing." 
 
-As Microsoft workers, we've been made complicit.
+As Microsoft workers, whilst we are proud of the efforts the company has made towards improving sustainability, we must do more.
 
 The Global Climate Strike is an opportunity for Microsoft to set an example as a leader in the tech industry. The company has taken many initiatives that focus on individual contributions, such as educating employees on composting, reducing waste, and making our [campus carbon neutral](https://blogs.microsoft.com/on-the-issues/2017/07/14/creating-carbon-free-headquarters-puget-sound-region/ ). There are also efforts to leverage artificial intelligence to solve specific and important climate-related problems. However, given the urgency and complexity of the crisis, these actions are not enough.
 
-If we want to make real impact, all of us need to mobilize, work together, and demand a fundamental change in Microsoft's priorities. It's clear that the tech industry is one of the main culprits behind our burning planet, both in consuming a great deal of fossil fuels through power hungry datacenters and supporting Big Oil companies. But we also have to recognize that we're equipped with so many resources that we can leverage to steer our industry and government in the right direction. 
+If we want to make real impact, all of us need to mobilize, work together, and demand a fundamental change in Microsoft's priorities. 
```

Comment:
I think the rest of this paragraph is an important supporting argument for the first sentence. You could explore wordsmithing it a bit more, but it really needs to remain more or less in place.